### DISCUSSION TITLE
How to represent muted notes?

### DISCUSSION THREAD

**Discussion by cecilios** [w3c-cg/musicxml#466] [Q&A] [answered]
The MusicXML format always tries to distinguish between elements used for sound information and elements used for notation information.

For display, to hide a note I understand that attribute `print-object="no"` will do the job. But for playback, what attribute is the analogous one to `print-object`? In summary, how to represent a muted note? 

My first idea was to use `<unpitched>`element but I soon discarded it as it does not imply muted.

**Comment by mdgood** [comment, marked as answer]
The [`<cue/>` element](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/cue/) indicates a silent note. If the note is played with a mute, the [`<mute>` element](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/mute/) is used, though that is not yet widely supported by MusicXML applications.

**Comment by cecilios** [comment]
Thank you! 
```
<note>
    ...
    <play>
        <mute>off</mute>
    </play>
</note>
```
is what I was looking for. Thank you.
